permissions_validators/src/private_blockchain/query.rs

```diff
@@ -283,6 +283,22 @@ impl<W: WorldTrait> IsAllowed<W, QueryBox> for OnlyAccountsDomain {
                     ))
                 }
             }
+            FindAssetDefinitionById(query) => {
+                let asset_definition_id = query
+                    .id
+                    .evaluate(wsv, &context)
+                    .map_err(|err| err.to_string())?;
+
+                if asset_definition_id.domain_id == authority.domain_id {
+                    Ok(())
+                } else {
+                    Err(format!(
+                        "Cannot access asset definition from a different domain. Asset definition domain: {}. Signers account domain {}.",
```

Comment:
there is just one signer, right?

```suggestion
                        "Cannot access asset definition from a different domain. Asset definition domain: {}. Signer's account domain {}.",
```